__tests__/unit/client/client.test.ts

```diff
@@ -89,20 +89,6 @@ describe("Client", () => {
       expect(axios.get).toHaveBeenCalledWith(`${url}/${contractName}`);
       expect(result).toBe(shieldContractAddress);
     });
-
-    test("Should return null if client app responds with status outside the successful range", async () => {
-      // Arrange
-      (axios.get as jest.Mock).mockRejectedValue(
-        new Error("Axios error at contract-address"),
-      );
-
-      // Act
-      const result = await client.getContractAddress(contractName);
-
-      // Assert
-      expect(axios.get).toHaveBeenCalledTimes(1);
-      expect(result).toBeNull();
-    });
```

Comment:
Removing since errors are now handled by an axios interceptor, so I will need to create a test for that